src/core/image.js

```diff
@@ -763,6 +798,10 @@ class PDFImage {
                 drawHeight,
                 forceRGB: true,
               });
+              if (isOffscreenCanvasSupported) {
```

Comment:
This would be a tiny bit clearer, and consistent with other code in this file, as follows:
```suggestion
              if (mustBeResized) {
```